connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -247,4 +271,29 @@ public void getUserInfo(
                     userInfoCallback);
         }
     }
+
+    private class LastSeenConfigStore {
```

Comment:
Why don't you prefer the TokenStore style?
E.g. rename this class to OperatorDiscoveryStore, and the methods set() and get()
Same for Wellknown config class.